src/transformers/onnx/features.py

```diff
@@ -345,6 +346,16 @@ class FeaturesManager:
             "token-classification",
             onnx_config_cls=RoFormerOnnxConfig,
         ),
+        "xlm": supported_features_mapping(
```

Comment:
After rebasing, please put this entry in alphabetical order

Reply:
Rebased with `main`, and changed  them according to alphabetical order, but few checks are failing due to changes from `Add YOLOS`. Can we ignore them or do I need to fix them?

Reply:
For some reason your branch has picked up changes on the test files for `yolos`. Perhaps the rebase didn't work fully as expected? Once those `yolos` files are removed, the tests should pass :)

Reply:
Thanks